src/components/Map/SearchPane/Drawer/DrawerContent/BusinessTypeEntities/BusinessTypeEntities.tsx

```diff
@@ -1,27 +1,32 @@
-import React, { useEffect } from 'react';
-import { Link } from 'react-router-dom';
-import { IonItem, IonText } from '@ionic/react';
+import React from "react";
+import { IonItem, IonText } from "@ionic/react";
+import { useNavigateBottomDrawer } from "../../drawerUtils";
 
 interface PageProps {
-    entitiesByBusinessType: any;
+  entitiesByBusinessType: any;
 }
 
 const BusinessTypeEntities: React.FC<PageProps> = (props) => {
+  const { navigateDrawer } = useNavigateBottomDrawer();
 
-    return (
-        <div style={{ height: '100%', marginTop: '4vh', overflow: 'visible' }}>
-            <IonText style={{ fontSize: 32, textDecoration: 'underline' }} color="primary" className="ion-text-capitalize ion-margin">{props.entitiesByBusinessType[0].business_type}</IonText>
-            {props.entitiesByBusinessType.map((entity:any,index:number) => {
-                return (
-                    <Link key={index} to={`/${entity.id}`} >
-                        <IonItem 
-                            className="search-suggestion-el"
-                        >{entity.name}</IonItem>
-                    </Link>
-                )
-            })}
-        </div>
-    );
+  return (
+    <div style={{ height: "100%", marginTop: "4vh", overflow: "visible" }}>
+      <IonText
+        style={{ fontSize: 32, textDecoration: "underline" }}
```

Comment:
Should this style be in CSS? Generally I'd only include styles in .tsx if they're critical to the operation of the interface element, or are manipulated by JS.

Let's tidy this up in a separate PR sometime